recipes/dask_traj/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "dask_traj" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "165f98016f59944cbd5960306ab26b69957f0abb14f8d4496d0dbe31977e59ad" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
```

Comment:
It looks like `scipy`, `distributed` and `pandas` are not actually used in the package . Note that you might want to update the upstream repo's `setup.py` to explicitly include `numpy` as a requirement